magneticod/setup.py

```diff
@@ -23,7 +23,8 @@ def read_file(path):
 
     install_requires=[
         "appdirs >= 1.4.3",
-        "bencoder.pyx >= 1.1.3"
+        "bencoder.pyx >= 1.1.3",
+        "humanfriendly"
```

Comment:
It doesn't make sens to specify the minimum version which is equal to first public release.